CHANGELOG.md

```diff
@@ -19,9 +19,24 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
     `/reconciliationReport` returns.  The user (dashboard) must read the value
     of `url` from the return to get the s3SignedURL and then download the report.
 
+### BREAKING CHANGES
+
+- **CUMULUS-2185** - RDS Migration Epic
+  - **CUMULUS-2191**
+    - Removed the following from the `@cumulus/api/models.asyncOperation` class in
+      favor of the added `@cumulus/async-operations` module:
+      - `start`
+      - `startAsyncOperations`
+
 ### Added
 
 - **CUMULUS-2185** - RDS Migration Epic
+  - **CUMULUS-2191**
+    - Updated cumuluss/async-operation task to write async-operations to the RDS
```

Comment:
This seems like it belongs under an "Updated" header?

Reply:
Hmm, given it has 'updated' in it... probably 😆

Reply:
61018962f

Reply:
And removed from here?